app/javascript/modules/listingDetailsLottery/ListingDetailsLottery.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from "react"
+import dayjs from "dayjs"
+import {
+  AppearanceSizeType,
+  AppearanceStyleType,
+  Button,
+  Heading,
+  t,
+} from "@bloom-housing/ui-components"
+import { RailsListing } from "../listings/SharedHelpers"
+import { isLotteryComplete } from "../../util/listingUtil"
+import { getLotteryBucketDetails } from "../../api/listingApiService"
+import { RailsLotteryBucketsDetails } from "../../api/types/rails/listings/RailsLotteryBucketsDetails"
+import { ListingDetailsLotteryModal } from "./ListingDetailsLotteryModal"
+
+export interface ListingDetailsLotteryProps {
+  listing: RailsListing
+}
+
+export const ListingDetailsLottery = ({ listing }: ListingDetailsLotteryProps) => {
+  const [lotteryBucketDetails, setLotteryBucketDetails] = useState<RailsLotteryBucketsDetails>()
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  useEffect(() => {
+    void getLotteryBucketDetails(listing.Id).then((lotteryBucketDetails) => {
+      setLotteryBucketDetails(lotteryBucketDetails)
+    })
+  }, [listing.Id])
+
+  return (
+    isLotteryComplete(listing) && (
+      <div className="border-b pt-4 text-center">
+        <Heading priority={4}>{t("listings.lottery.lotteryResults")}</Heading>
```

Comment:
we'll want to add some margin here to match angular
```suggestion
        <Heading className="mb-16" priority={4}>{t("listings.lottery.lotteryResults")}</Heading>
```